### PR TITLE
Text changes april2023 onwards

### PR DESCRIPTION
**Pull request by manishjain999**
Updated texts 24 June 2023

### CODE REVIEW

File: config/config-es.json [hunk 1]
```diff
@@ -60,12 +60,12 @@
         "startNonTimerBreakActivityTitleText": "Es hora de un microdescanso. Hagamos <ACTIVITY> para mantener la mente y el cuerpo despiertos.",
         "completionRequirementsText": "Completar después: <REQUIREMENTS>",
         "titleCheckList": "Lista de Control",
-        "activitySkippedWhatNextSoundText": "<ACTIVITY> omitido. ¿Adivina qué sigue?",
-        "activityFinishedWhatNextSoundText": "<ACTIVITY> finalizado. ¿Adivina qué sigue?",
-        "activitySkippedNextActivitySoundText": "<ACTIVITY> omitido. Lo siguiente es <NEXTACTIVITY>.",
-        "activityFinishedNextActivitySoundText": "<ACTIVITY> finalizado. Lo siguiente es <NEXTACTIVITY>.",
-        "activitySkippedUnlockNowSoundText": "<ACTIVITY> omitido. Te has ganado tu tiempo de pantalla.",
-        "activityFinishedUnlockNowSoundText": "<ACTIVITY> finalizado. Te has ganado tu tiempo de pantalla."
+        "activitySkippedWhatNextSoundText": "<ACTIVITY> skipped. Guess Whats next?[NEEDSTRANSLATION]",
```
Comment (jeznag) [inline config/config-es.json:63]: Merge conflicts

Comment (jeznag) [reply config/config-es.json:63]: This was already translated

Comment (manishjain999) [reply config/config-es.json]: Fixed it. Could you review please.

File: config/config.json [hunk 2]
```diff
@@ -909,7 +919,8 @@
         "titleUrlSafeText": "Safe site according to AI (probability <PERCENTAGE>%)",
         "reasonText": "Reason: <REASON>",
         "buttonBadDecisionText": "Bad decision",
-        "checkboxDonotShowText": "Don't show this anymore"
+        "checkboxDonotShowText": "Don't show this anymore",
+        "progressInfoText": "Here is the progress while you are teaching Focus Bear to keep allowing sites/apps but within a few days, you'll have taught Focus Bear what apps and websites are productive."
```
Comment (jeznag) [inline config/config.json:923]: You're teaching Focus Bear what sites and apps are productive for you. It takes a bit of effort at first but within a few days, you'll have taught Focus Bear what apps and websites are productive.

File: config/config-es.json [hunk 3]
```diff
@@ -910,7 +920,8 @@
         "titleUrlSafeText": "Sitio seguro según IA (probabilidad <PERCENTAGE>%)",
         "reasonText": "Motivo: <REASON>",
         "buttonBadDecisionText": "Mala decisión",
-        "checkboxDonotShowText": "No volver a mostrar esto"
+        "checkboxDonotShowText": "No volver a mostrar esto",
+        "progressInfoText": "Here is the progress while you are teaching Focus Bear to keep allowing sites/apps but within a few days, you'll have taught Focus Bear what apps and websites are productive.[NEEDSTRANSLATION]"
```
Comment (jeznag) [inline config/config-es.json:924]: You're teaching Focus Bear what sites and apps are productive for you. It takes a bit of effort at first but within a few days, you'll have taught Focus Bear what apps and websites are productive.

File: config/config-es.json [hunk 4]
```diff
@@ -1205,4 +1217,4 @@
         "downloadingUpdate": "Descargando Actualización",
         "downloadProgress": "Descargando Actualización: {0} de {1}"
     }
-}
+}
```
Comment (jeznag) [inline config/config-es.json:1220]: trailing new line

**Review by jeznag — APPROVED** [review]

**Pull request merged**